libs/payments/src/lib/payment-provider/bexioPaymentProvider.ts

```diff
@@ -0,0 +1,237 @@
+import Bexio, {ContactsStatic, InvoicesStatic} from 'bexio'
+import ContactSearchParameters = ContactsStatic.ContactSearchParameters
```

Comment:
what's this syntax?

Reply:
An import Bexio is default export and ContactsStatic, InvoicesStatic are others. The secound line is a way to import  a type or an interface.